pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -247,9 +240,9 @@ func (vm *Windows) ConfigureCNI(configFile string) error {
 	configureCNICmd := remoteDir + "wmcb.exe configure-cni --cni-dir=\"" +
 		cniDir + " --cni-config=\"" + cniConfigDest
 
-	_, stderr, err := vm.Run(configureCNICmd, true)
-	if err != nil || len(stderr) > 0 {
-		log.Info("CNI configuration failed", "CNI configuration command", configureCNICmd, "stderr", stderr, "err", err)
+	out, err := vm.RunOverSSH(configureCNICmd, true)
+	if err != nil {
+		log.Info("CNI configuration failed", "CNI configuration command", configureCNICmd, "output", out)
```

Comment:
Shouldn't we capture err as well?

Reply:
Sorry I missed that. I will add that back.